boards/STM32F746IGT6-WaveShare-Core7XXI.md

```diff
@@ -84,11 +84,7 @@ title: "STM32F746IGT6 - WaveShare Core7XXI"
     <tr>
         <td rowspan="4"><b>Connectivity</b></td>
         <td>Headers</td>
-        <td>2x 2x27 male dupont (2.54mm)</td>
-    </tr>
-    <tr>
-        <td>Specific</td>
-        <td>Serial (1x4 male dupont (2.54mm))</td>
```

Comment:
Do not remove the full row, since this will break the table layout. You can simply put "None" in there.